src/main/java/com/superdevs/HealthOMeter/entity/User.java

```diff
@@ -53,6 +66,22 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    public LocalDateTime getLastActivity() {
+        return lastActivity;
```

Comment:
Imo shouldn't we be a little more specific? Like: LocalDate.now or something close to that?

Reply:
Hard to tell. @Nebres84 - can I kindly ask for your recommendations?